### DISCUSSION TITLE
Search Across Platforms

### DISCUSSION THREAD

**Discussion by Budlyte** [rommapp/romm#642] [Q&A]
Is there currently a way to search the entire library, across all platforms, that I am missing?  Suppose I have completely legal copies of Final Fantasy 1 thru 5 on GBA, 6 on SNES, and 7 on PC, but wanted to search "Final Fantasy" to find a specific one without hopping around Platforms.

**Comment by zurdi15** [comment]
It will be in the next release ;) https://github.com/zurdi15/romm/pull/612
